spyderlib/dependencies.py

```diff
@@ -80,7 +79,6 @@ def check(modname):
 
 def status():
     """Return a complete status of Optional Dependencies"""
-    global DEPENDENCIES

```

Comment:
Because at first I wanted to use a list for the dependencies so I looked how dependencies.status() was coded. This is a simple change to clean up the codebase a bit so I included it anyway. global is needed only when defining new variables, not when using them.